engine/pkg/orm/client.go

```diff
@@ -621,6 +636,41 @@ func (c *metaOpsClient) QueryResourcesByExecutorID(ctx context.Context, executor
 	return resources, nil
 }
 
+func (c *metaOpsClient) DeleteResourcesByExecutorID(ctx context.Context, executorID string) error {
+	tx := c.db.WithContext(ctx).Where("executor_id = ?", executorID).
+		Delete(&resourcemeta.ResourceMeta{})
+	if tx.Error != nil {
```

Comment:
seems we can directly return cerrors.ErrMetaOpFail.Wrap(tx.Error)

Reply:
This will cause problems for the caller, because `cerrors.ErrMetaOpFail.Wrap(tx.Error)` returns `(*errors.Error)(nil)` instead of `nil` if `tx.Error == nil`.